README.md

```diff
@@ -162,9 +154,52 @@ To provide Keystone credentials and configure the agent using auto-detection run
 
 For more parameters, please see [Monasca Agent Documentation](https://github.com/openstack/monasca-agent/blob/master/docs/Agent.md#explanation-of-primary-monasca-setup-command-line-parameters).
 
-This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above.
+This will create and run a new service file `/etc/systemd/system/monasca-agent.service`
+with the configuration set as per the arguments mentioned above.
+
+### Uninstalling Metrics agent
+Stop the services:
+```
+systemctl stop monasca-agent
+systemctl disable monasca-agent
+```
+
+In the following description, `[target_dir]` is the target directory specified
+at the time of running `monasca-agent-[version].run`.
 
-### Log agent
+Delete created files:
+```
+sudo rm -rf [target_dir]
+sudo rm -rf /etc/monasca/agent/
+sudo rm -rf /etc/sudoers.d/mon-agent/
```

Comment:
`mon-agent` is file so please change as below.
```
sudo rm -f /etc/sudoers.d/mon-agent
```

Reply:
Done